core/modules/drr.cc

```diff
@@ -0,0 +1,392 @@
+#include "drr.h"
+
+#include <rte_ether.h>
+#include <rte_ip.h>
+#include <rte_udp.h>
+#include <time.h>
+#include <glog/logging.h>
+
+#include <string>
+#include <cmath>
+#include <iostream>
+#include <fstream>
+
+
+#include "../mem_alloc.h"
+
+int RoundToPowerTwo(uint32_t v) {
```

Comment:
This is defined in `core/utils/common.h` as `align_ceil_pow2`. If fields like `max_flow_queue_size_` can be changed to uint64_t, I recommend using it instead.